ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -241,14 +241,21 @@ wan_path_groups:
     ipsec: false
     # TODO remove one once auto-id is implemented - for now required in schema
     id: 100
+    dps_keepalive:
+      interval: 300
   - name: INET
     id: 101
   - name: LTE
     id: 102
   - name: Equinix
     id: 103
+    dps_keepalive:
+      interval: 100
+      failure_threshold: 12
```

Comment:
This is not rendered anywhere so the test is a bit moot

Reply:
Thanks, shifted this case to LTE path group, now it is rendered on cv-pathfinder-edge.